src/librustc_resolve/build_reduced_graph.rs

```diff
@@ -314,7 +314,7 @@ impl<'a> Resolver<'a> {
                             Ident::new(kw::SelfLower, new_span)
                         ),
                         kind: ast::UseTreeKind::Simple(
-                            Some(Ident::from_str_and_span("__dummy", new_span).gensym()),
+                            Some(Ident::new(kw::Underscore, new_span)),
```

Comment:
Hmm, this code is buggy (https://github.com/rust-lang/rust/issues/61826), but this change shouldn't make it more buggy since the import is `ty::Visibility::Invisible`.